Software/SoftwareLaunchers.md

```diff
@@ -8,16 +8,19 @@ Join our [discord](https://discord.gg/8nzHYhVUQS) or use the issues.<br><br>
 
 ## Launchers
 
-| Name | Supported Loaders | Supported Versions | Description | Rating |
```

Comment:
I also removed the rating column, because it seemed arbitrary and I thought the other columns and the software's main page are what should give other people an idea of it's quality.

Reply:
I think the rating column should stay because it makes it easy for people to find a good launcher. It's just a way for us to convey our experience with the launcher. Plus, there's no harm in keeping it.

Reply:
the problem is that you cannot quantify entirely the "goodness" of the launcher into a single number accurately and different people will look at different qualities when thinking about how good a launcher is. for example, the fact that FTB & curseforge are running on the bloated overwolf platform makes them very heavy compared to, for instance, multimc & just the fact alone that one must install overwolf to use them makes them a lot less desirable. overwolf will default to run at startup and all the other typical annoyances with similar softwares like it. badlion has many trackers in its launcher that will upload analytics without notifying the user & without a complete opt-out option. multimc is very light and has completely modular instances & many options for customisation and adding multiple accounts. it is by far the "best" in terms of features, modpack coverage, and efficiency. the caveat is it is slightly less user friendly than a simpler launcher, but this is the problem I am talking about, as different people seek different qualities in a launcher, a single number arbitrary rating really doesnt mean much in reality. there's also the fact that all the launchers (besides the very very old ones and FTB App) are within the 7-9 score range, further diluting the meaningfulness of such a rating. I also really think that if people care enough to look into launchers, they will surely come to their own conclusions and as such the rating would be pointless as well as arbitrary. it just doesnt really make sense to me and it just reduces the quality of the article in my opinon.

Reply:
also, very great work on the java article, it has things even I didnt know about and covers all the important things and I think you've done a great job with it.

Reply:
> the problem is that you cannot quantify entirely the "goodness" of the launcher into a single number accurately and different people will look at different qualities when thinking about how good a launcher is. for example, the fact that FTB & curseforge are running on the bloated overwolf platform makes them very heavy compared to, for instance, multimc & just the fact alone that one must install overwolf to use them makes them a lot less desirable. overwolf will default to run at startup and all the other typical annoyances with similar softwares like it. badlion has many trackers in its launcher that will upload analytics without notifying the user & without a complete opt-out option. its anti-cheat also seems a lot more invasive than many would be alright with. multimc is very light and has completely modular instances & many options for customisation and adding multiple accounts. it is by far the "best" in terms of features, modpack coverage, and efficiency. the caveat is it is slightly less user friendly than a simpler launcher, but this is the problem I am talking about, as different people seek different qualities in a launcher, a single number arbitrary rating really doesnt mean much in reality. there's also the fact that all the launchers (besides the very very old ones and FTB App) are within the 7-9 score range, further diluting the meaningfulness of such a rating. I also really think that if people care enough to look into launchers, they will surely come to their own conclusions and as such the rating would be pointless as well as arbitrary. it just doesnt really make sense to me and it just reduces the quality of the article in my opinon.

You're right, different people seek different features of different launchers. The aim of this project is to show all the options to the people and not tell them what's best, as that can vary from person to person. I thought ratings would help people find a good launcher easier, but the ratings do seem quite arbitrary and don't really mean anything.

Reply:
> also, very great work on the java article, it has things even I didnt know about and covers all the important things and I think you've done a great job with it.

Thanks! I myself learnt a few things while researching to make this article.